src/queries/__tests__/host-component-names.test.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { Text, TextInput } from 'react-native';
+import { render, screen, configure } from '../../pure';
+
+describe('host component names', () => {
```

Comment:
in this repo, we don't use top level describe except if there is several of them at the root of the file